NOTICE.txt

```diff
@@ -3581,7 +3581,7 @@ SOFTWARE.
 
 --------------------------------------------------------------------
 Dependency: golang.org/x/crypto
-Revision: 9419663f5a44be8b34ca85f08abc5fe1be11f8a3
+Revision: 9f005a07e0d31d45e6656d241bb5c0f2efd4bc94
```

Comment:
Could you update the revision of the dependencies in an separate PR? This would make sure the version change does not break other things and it would make this PR much smaller.

Reply:
Agree, I've marked as as a TODO in the PR description.